InfoLiga/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "infoliga",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node index.js"
+  },
+  "author": "Miguel Carballo Garcia",
+  "license": "ISC",
+  "devDependencies": {
+    "eslint": "^4.5.0",
+    "eslint-config-5minds": "^0.1.0",
```

Comment:
du brauchst nur die 5minds config als devDependency - und zusätzlich eine .npmrc-Datei - findest du hier in der README: https://www.npmjs.com/package/eslint-config-5minds